src/app/application-form/application-form.component.ts

```diff
@@ -221,6 +225,12 @@ export class ApplicationFormComponent implements OnInit {
       .get('downPayment') as FormControl<number>;
   }
 
+  get percentage() {
+    return this.applicationForm
+      .get('loanForm')!
```

Comment:
Don't we already have a `this.loanForm` somewhere in the code?
```suggestion
      return this.loanForm.get('percentage');
```